lib/docassemble/docassemble_api_interface.js

```diff
@@ -222,7 +222,7 @@ da_i.loop_to_create_project = async function( creation_options ) {
 
     } catch ( error ) {
       // TODO: Make a `normalize_ajax_responses_and_errors()`?
-      let response = error.response;
+      let response = error;
```

Comment:
This is unrelated, right? A different bug?

Reply:
Yes, different bug but I did need to fix it to test. I think it's related to those 400 errors we've been seeing with Caroline and elsewhere recently.